e2e/playwright/code-pane-and-errors.spec.ts

```diff
@@ -179,7 +179,7 @@ extrude001 = extrude(sketch001, length = 5)`
     await page.setBodyDimensions({ width: 1200, height: 500 })
     await homePage.goToModelingScene()
 
-    await scene.settled(cmdBar)
```

Comment:
We can't use `scene.settled` anymore on bad kcl since the Start Sketch button will never be enabled @franknoirot. Guess I could update the fixture? 👀